Sources/Sentry/SentrySDK.m

```diff
@@ -61,6 +61,11 @@ + (void)setCurrentHub:(nullable SentryHub *)hub
     return currentHub.scope.span;
 }
 
++ (BOOL)isEnabled
+{
+    return currentHub != nil;
```

Comment:
`h`: The currentHub is not `nil` when you call `SentrySDK.currentHub` once, as we initialize a hub without a client. We also need to check if the hub has a client. Currently, even if you never call `SentrySDK.start` `SentrySDK.isEnabled` would return `true` if you do this

```swift
SentrySDK.captureMessage("hello")
XCTAssertFalse(XCSentrySDK.isEnabled) // would be true
```